go/vt/topo/topoproto/tablet_test.go

```diff
@@ -70,3 +72,54 @@ func TestParseTabletAlias(t *testing.T) {
 		}
 	}
 }
+
+func TestTabletsInList(t *testing.T) {
+	tablet1 := &topodatapb.Tablet{
+		Alias: &topodatapb.TabletAlias{
+			Cell: "zone-1",
+			Uid:  1,
+		},
+	}
+	tablet2 := &topodatapb.Tablet{
+		Alias: &topodatapb.TabletAlias{
+			Cell: "zone-1",
+			Uid:  2,
+		},
+	}
+	tablet3 := &topodatapb.Tablet{
+		Alias: &topodatapb.TabletAlias{
+			Cell: "zone-1",
+			Uid:  3,
+		},
+	}
+	testcases := []struct {
+		name       string
+		tablet     *topodatapb.Tablet
+		allTablets []*topodatapb.Tablet
+		isInList   bool
+	}{
+		{
+			name:       "empty list",
+			tablet:     tablet1,
+			allTablets: nil,
+			isInList:   false,
+		}, {
+			name:       "tablet in list",
+			tablet:     tablet2,
+			allTablets: []*topodatapb.Tablet{tablet1, tablet2, tablet3},
+			isInList:   true,
+		}, {
+			name:       "tablet not in list",
+			tablet:     tablet1,
+			allTablets: []*topodatapb.Tablet{tablet2, tablet3},
+			isInList:   false,
+		},
+	}
+
+	for _, testcase := range testcases {
```

Comment:
suggestion -- run these test cases in parallel with:

```go
func TestBlahBlah(t *testing.T) {
    t.Parallel()

    for _, testcase := range testcases {
        testcase := testcase // prevent loop shadowing
        t.Run(testcase.name, func(t *testing.T) {
            t.Parallel()
            // rest of test
        })
    }
}
```

Reply:
Fixed